test.py

```diff
@@ -785,6 +785,7 @@ def _reset(self) -> None:
         self.is_after_test_ok = False
         self.args = [
             "-s",  # don't capture print() output inside pytest
+            "--log-level=DEBUG",   # Capture logs
```

Comment:
Can you explain which logs exactly this is enabling? Because there are already some logs from all sources (the harness and the test).